From fe1a3b48bbb41190d1ea41b9f6ce663c83748ca5 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 15 Aug 2022 18:27:00 +0000
Subject: [PATCH] IWYU: add vector for std::vector in browser_finder.h

Bug: 957519
Change-Id: Icd75f2d982cf8c24b5d5cae05a4d5a397cc7e2df
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3830011
Reviewed-by: Dana Fried <dfried@chromium.org>
Commit-Queue: Dana Fried <dfried@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1035128}
---
 chrome/browser/ui/browser_finder.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/ui/browser_finder.h b/chrome/browser/ui/browser_finder.h
index f885be017e422..ad7a184fa5277 100644
--- a/chrome/browser/ui/browser_finder.h
+++ b/chrome/browser/ui/browser_finder.h
@@ -6,6 +6,7 @@
 #define CHROME_BROWSER_UI_BROWSER_FINDER_H_
 
 #include <stddef.h>
+#include <vector>
 
 #include "ui/display/types/display_constants.h"
 #include "ui/gfx/native_widget_types.h"
-- 
2.37.2

